pages/linux/dnf-group.md

```diff
@@ -0,0 +1,24 @@
+# dnf group
+
+> Manage groups of related DNF packages on Fedora-based systems.
```

Comment:
Personally, I like how it's explained in the dnf man page:

> Groups  are  virtual  collections of packages. DNF keeps track of groups that the user selected ("marked") installed and can manipulate the comprising packages with simple commands.

And if I'm being scholastic here, "DNF packages" really isn't a thing; rather it's all just "RPM packages". But I think it's unnecessarily pedantic for this page anyways.

So how about this?

```suggestion
> Manage virtual collections of packages on Fedora-based systems.
```